src/clients/feed/feed.ts

```diff
@@ -53,42 +53,39 @@ class Feed {
     this.broadcaster = new EventEmitter({ wildcard: true, delimiter: "." });
     this.defaultOptions = options;
 
-    // Try and connect to the socket
-    this.apiClient.socket.connect();
-
     this.channel = this.apiClient.socket.channel(
       `feeds:${this.userFeedId}`,
       this.defaultOptions,
     );
+
+    this.channel.on("new-message", (resp) => this.onNewMessageReceived(resp));
```

Comment:
Moved this from the `listenToUpdate` function given that it only needed to be bound once